pkg/placement/raft/server.go

```diff
@@ -206,57 +216,6 @@ func (s *Server) Raft() *raft.Raft {
 	return s.raft
 }
 
-// JoinInitialCluster joins the initial peers into raft cluster.
```

Comment:
We do not need this code because `bootstrapConfig` can set initial peer servers during bootstrap.